pr2eus/pr2-interface.l

```diff
@@ -81,6 +92,17 @@
        (setq joint-action-enable nil)
        (ros::ros-warn "~A is not respond, pr2-interface is disabled" action)
        (return)))
+   ;; install interruption handler
+   (if *enable-roseus-resume*
+     (progn
+       (warning-message 3 "Installing interruption handler...~%")
+       (roseus_resume::install-interruption-handler self
```

Comment:
I am not familer with roseus_resume API, but define
```
(:install-roseus-resume-interruption-handler (args)
 (if *enable-roseus-resume* (progn ...) (progn ....)
```
 in  robot-interface or robot-move-base-interface and add 
```
(send self :install-roseus-resume-interaption-handler (list r-gripper-action l-gripper-action))
```
in pr2-interface.l

- I am qurious why we do not have to specity joint-trajectory-action?
- if we need to install all actionclient, may be we try to find them from program code

Reply:
I think that adding a `:install-roseus-resume-interruption-handler` method in the robot interface with internal checking is a valid approach for portability.

> I am curious why we do not have to specify joint-trajectory-action?

We already have access to joint trajectory actions via the `controller-actions` slot in the robot interface. So we only need to pass the "additional" actions in initialization.
https://github.com/Affonso-Gui/roseus_resume/blob/eus10/euslisp/interruption-handler.l#L51-L58

> if we need to install all actionclient, may be we try to find them from program code

It is theoretically possible to loop through all slots/bindings/symbols and register all instances of `ros::simple-action-client`.

Reply:
can we add additional controllers into `controller-actions`?
if not, we can make a new slot for additional controllers such as `additional-controller-actions`.
If we can do this, we no more need to change codes in `fetcheus` or `baxtereus`, we only need to change `pr2eus/robot-interface.l`.

Reply:
also, we need to change the order.
`(send-super* :init args)` is normally called at the first of `:init`, and we need to move `:add-controller` before `send-super`.